api/migrations/20200331220053-create-calendar-event.js

```diff
@@ -46,6 +46,16 @@ module.exports = {
         allowNull: false,
         type: Sequelize.DATE,
       },
+      is_public: {
```

Comment:
You have to create a new migration and add the columns there, this is completely wrong.

Reply:
Ohh sure.. What should i name the new migration file?

Reply:
add_is_public_and_is_open_to_calendar_event

Reply:
Done!